content/altinn-studio/v10/develop-a-service/data/restricted-data/_index.en.md

```
@@ -19,7 +19,7 @@ You can read more about the concept [here](/en/altinn-studio/v8/concepts/data-mo
 ## Configuring Maskinporten
 You must configure Maskinporten to allow the app to perform actions on behalf of the service owner.
 
-You can find a detailed guide on that setup [here](/en/altinn-studio/v8/guides/integration/maskinporten/).
+You can find a detailed guide on that setup [here](/en/altinn-studio/v10/develop-a-service/integration/maskinporten/).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)